docker/entrypoint-poolimage.sh

```diff
@@ -17,9 +17,15 @@ fi
 
 # Disabling coredumps by default in the shell where zrepl runs
 if [ -z "$ENABLE_COREDUMP" ]; then
+	echo "Disabling dumping core"
 	ulimit -c 0
+else
+	echo "Enabling coredumps"
+	ulimit -c unlimited
+	cd /var/openebs/sparse || exit
```

Comment:
We should return non-zero value if sparse directory doesn't exist.

Reply:
$ help exit
exit: exit [n]
    Exit the shell.
    
    Exits the shell with a status of N.  If N is omitted, the exit status
    is that of the last command executed.

Reply:
it returns error 2 in this case 'no such file or directory'
```
vitta@vitta-laptop:~$ sudo docker run --env ENABLE_COREDUMP=1 --mount 'type=bind,src=/home/vitta/Desktop,dst=/home/core' openebs/cstor-pool:ulimit5
Enabling coredumps
at call_exit..
exit code: 0
reference:  /usr/local/bin/entrypoint-poolimage.sh
/usr/local/bin/entrypoint-poolimage.sh: 25: cd: can't cd to /var/openebs/sparse
vitta@vitta-laptop:~$ echo $?
2
```